migrations/20210401161450_add_notifications_table.js

```diff
@@ -0,0 +1,19 @@
+
+exports.up = function(knex) {
+  return knex.schema
+    .createTable('notifications', table => {
+      table.uuid('id').primary().defaultTo(knex.raw('uuid_generate_v4()'));
+      table.string('to').notNull();
+      table.string('name').notNull();
+      table.string('subject').notNull();
+      table.text('html').notNull();
+      table.string('identifier').notNull();
+      table.dateTime('completed').nullable();
+      table.dateTime('deleted');
```

Comment:
Should this also be `nullable()` (maybe that's the default?)

Reply:
It is the default.